containers/Singularity/rocky-8.def

```diff
@@ -14,4 +14,11 @@ Include: dnf
     shadow-utils rsyslog tzdata util-linux words zlib tar less \
     gzip which util-linux openssh-clients openssh-server dhclient \
     pciutils vim-minimal shadow-utils strace cronie crontabs cpio \
-    wget rocky-release ipmitool yum NetworkManager
\ No newline at end of file
+    wget rocky-release ipmitool yum NetworkManager network-scripts
+
+    systemctl disable NetworkManager
+    systemctl disable NetworkManager-dispatcher
+    systemctl disable NetworkManager-wait-online
+    systemctl enable network-scripts
```

Comment:
oops, I mistyped this originally, should be the following:
`systemctl enable network`

Not a native systemd service, this creates SysV style init script links.